### DISCUSSION TITLE
Error: not enough values to unpack

### DISCUSSION THREAD

**Discussion by marisalim** [nih-cfde/published-documentation#150] [Q&A] [answered]
I ran `cfde-submit run ./Github/c2m2/draft-C2M2_example_submission_data/HMP__sample_C2M2_Level_1_bdbag.contents -v` and after checking the status with `cfde-submit status`, I got this output/error. How do I fix it?
```
Running on service 'prod'
Running on service 'prod'

Status of CFDE Submission (Flow ID 081ac022-ca14-4c41-b808-b33ff2501bb1)
This instance ID: 9a782add-04c7-41ee-b8c2-116838573832

This Flow has completed.
Your CFDE submission () failed to ingest into DERIVA with this error:
Error ingesting to DERIVA: not enough values to unpack (expected 2, got 1)
```

**Comment by ACharbonneau** [comment, marked as answer]
This error likely means that you tried to use a DCC ID that was not recognized. DCC IDs are two parts, and you submitted only one. 

**Possible solutions:**

1. Find your correct DCC ID

Use [these instructions](https://docs.nih-cfde.org/en/latest/cfde-submit/docs/#find-your-dccs-identifier) to find your DCC ID. Be sure to use the full ID in the tool, it should look something like `cfde_registry_dcc:hmp`. IDs are space and case sensitive.

**Reply by ACharbonneau** [reply]
This has been fixed. As of 3/30/2021 the submission tool now takes DCC IDs in the form of `cfde_registry_dcc:hmp` or just `hmp`
